stories/settings.stories.js

```diff
@@ -47,12 +51,11 @@ storiesOf( 'Settings', module )
 	} )
 	.add( 'Connected Services', () => {
 		global.googlesitekit = settingsData;
-		global.googlesitekit.setupComplete = true;
```

Comment:
Why were these removed?

Reply:
Oh, good catch, these changes are leftovers related to something I temporarily changed elsewhere but then reverted. Will change this back.